typescript-test-samples/appsync-cognito-lambda-aleios/metadata.json

```diff
@@ -8,7 +8,7 @@
   "framework": "CDK",
   "services": ["appsync", "lambda", "cognito"],
   "git_repo_url": "https://github.com/aws-samples/serverless-test-samples",
-  "pattern_source": "aleios",
+  "pattern_source": "Customer",
```

Comment:
Could this be 'Partner'?

Reply:
Updated type to Partner and added pattern_source_name as Aleios